### DISCUSSION TITLE
Detected by StackPath

### DISCUSSION THREAD

**Discussion by pogarek** [ultrafunkamsterdam/undetected-chromedriver#1140] [General]
HI
I scrape cell tracker for part of data. Recently I got detected there, after w few clicks with the code, which was working for months..
They are protected by/with stackpath.com . 

Any idea how to  get undetected again?

**Comment by torzak** [comment]
Same for me... @pogarek Have you managed to find a solution?

Sometimes, I close and start a new session, which allows me to proceed. However, after a while, everything stops working and I end up being blacklisted. **It's not my IP address that's blacklisted because when I use a clean browser, I'm not blocked**.

**Reply by pogarek** [reply]
Unfortunately not.. :-(
